packages/mongo-nav/src/types.ts

```diff
@@ -220,60 +220,78 @@ export interface OnPremInterface {
   enabled?: boolean;
 }
 
-export interface URLSInterface {
-  userMenu?: {
-    cloud?: {
-      userPreferences: string;
-      organizations: string;
-      invitations: string;
-      mfa: string;
-    };
-    university?: {
-      universityPreferences: string;
-    };
-    support?: {
-      userPreferences: string;
-    };
-    account?: {
-      homepage?: string;
-    };
-    logout?: string;
+interface UserMenuURLS {
+  cloud?: {
+    userPreferences: string;
+    organizations: string;
+    invitations: string;
+    mfa: string;
   };
-  mongoSelect?: {
-    viewAllProjects?: string;
-    viewAllOrganizations?: string;
-    newProject?: string;
-    orgSettings?: string;
+  university?: {
+    universityPreferences: string;
   };
-  orgNav?: {
-    leaf?: string;
-    settings?: string;
-    accessManager?: string;
-    support?: string;
-    billing?: string;
-    allClusters?: string;
-    admin?: string;
+  support?: {
+    userPreferences: string;
   };
-  projectNav?: {
-    settings?: string;
-    accessManager?: string;
-    support?: string;
-    integrations?: string;
-    alerts?: string;
-    activityFeed?: string;
-    invite?: string;
-    realm?: string;
-    charts?: string;
-  };
-  onPrem?: {
-    profile?: string;
-    mfa?: string;
-    personalization?: string;
-    invitations?: string;
-    organizations?: string;
-    publicApiAccess?: string;
-    featureRequest?: string;
+  account?: {
+    homepage?: string;
   };
+  logout?: string;
+}
+
+interface MongoSelectURLS {
+  viewAllProjects?: string;
+  viewAllOrganizations?: string;
+  newProject?: string;
+  orgSettings?: string;
+}
+
+interface OrgNavURLS {
+  leaf?: string;
+  settings?: string;
+  accessManager?: string;
+  support?: string;
+  billing?: string;
+  allClusters?: string;
+  admin?: string;
+}
+
+interface ProjectNavURLS {
+  settings?: string;
+  accessManager?: string;
+  support?: string;
+  integrations?: string;
+  alerts?: string;
+  activityFeed?: string;
+  invite?: string;
+  realm?: string;
+  charts?: string;
+}
+
+interface OnPremURLS {
+  profile?: string;
+  mfa?: string;
+  personalization?: string;
+  invitations?: string;
+  organizations?: string;
+  publicApiAccess?: string;
+  featureRequest?: string;
+}
+
+export interface URLSProp {
+  userMenu?: UserMenuURLS;
+  mongoSelect?: MongoSelectURLS;
+  orgNav?: OrgNavURLS;
+  projectNav?: ProjectNavURLS;
+  onPrem?: OnPremURLS;
+}
+
+export interface URLSDefaults {
```

Comment:
Defaults implies there's values associated with it. Also we use this everywhere internally...I wonder if we should just call it `URLS`.

Reply:
kk

Reply:
updated